rocketpy/AeroSurface.py

```diff
@@ -1892,3 +1883,115 @@ def allInfo(self):
         self.aerodynamicInfo()
 
         return None
+
+
+class RailButtons(AeroSurface):
+    """Class that defines a generic rail button.
+
+    Attributes
+    ----------
+    RailButtons.buttons_distance : int, float
+        Distance between the highest and the lowest rail button in meters.
```

Comment:
Highest means "the one with the greatest vertical extent". 
I do not think that you wanted to mean the height of the rail button, but that is what it sounds.